boilerplate/storage_service/models.py

```diff
@@ -13,3 +13,14 @@ class Product(models.Model):
     def save(self, *args, **kwargs):
         self.full_clean()
         return super(Product, self).save(*args, **kwargs)
+
+class CompleteProduct(Product):
+    name = models.CharField(max_length=64)
+    base_price = models.IntegerField()
+    description = models.CharField(max_length=128)
+    weight = models.FloatField()
+    category = models.CharField(max_length=32)
+
+    def save(self, *args, **kwargs):
+        self.full_clean()
+        return super(Product, self).save(*args, **kwargs)
```

Comment:
maybe this should call `super(CompleteProduct, ...)` rather than `super(Product, ...)`